<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>613</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Terrain Navigator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Experiment">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Experiment</string>
       </attribute>
       <widget class="QPushButton" name="prevSubTask">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Prev</string>
        </property>
       </widget>
       <widget class="QPushButton" name="nextSubTask">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QLabel" name="task_label">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Switch</string>
        </property>
       </widget>
       <widget class="QWidget" name="task1_widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>231</width>
          <height>161</height>
         </rect>
        </property>
        <widget class="QPushButton" name="moveTerDown">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>81</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
        <widget class="QPushButton" name="moveTerUp">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>110</y>
           <width>81</width>
           <height>27</height>
          </rect>
         </property>
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>121</width>
           <height>71</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>LandscapeA</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LandscapeB</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LandscapeC</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="task2_widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>221</width>
          <height>261</height>
         </rect>
        </property>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>201</width>
           <height>241</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>131</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Information</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="subtask_index_label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>1/1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Scene">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Scene</string>
       </attribute>
       <widget class="QLabel" name="background_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="skybox_checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Skybox</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadSkybox_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>151</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Skybox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="wirefr_checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loadTexture_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Texture</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="texture_checkBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Texture</string>
        </property>
       </widget>
       <widget class="QSlider" name="bgR_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>25</width>
          <height>81</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="bgG_box">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>25</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="bgB_box">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>25</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="fps_checkBox">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>210</y>
          <width>61</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Lights">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Lights</string>
       </attribute>
       <widget class="QLabel" name="light_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>62</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="enableLight_box">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="lposition_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
       <widget class="QLabel" name="ambient_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Ambient </string>
        </property>
       </widget>
       <widget class="QLabel" name="diffuse_label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>200</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Diffuse</string>
        </property>
       </widget>
       <widget class="QLabel" name="specular_label">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>61</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Specular</string>
        </property>
       </widget>
       <widget class="QLabel" name="lshininess_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Shininess</string>
        </property>
       </widget>
       <widget class="QSlider" name="mshininess">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>310</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ambR_box">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ambG_box">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="ambB_box">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="difR_box">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="difB_box">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="difG_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="spcR_box">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="spcG_box">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="spcB_box">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>220</y>
          <width>21</width>
          <height>81</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lPosZ_box">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lPosY_box">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lPosX_box">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
       <widget class="QCheckBox" name="enableGLSL_box">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>93</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>GLSL</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="lPosW_box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>91</width>
          <height>25</height>
         </rect>
        </property>
        <property name="minimum">
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Viewer" name="viewer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionLoad_Experiment"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionSave_Config"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load Terrain</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save screenshot</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit this viewer</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionLoad_Experiment">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Experiment</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header>QGLViewer/qglviewer.h</header>
  </customwidget>
  <customwidget>
   <class>Viewer</class>
   <extends>QGLViewer</extends>
   <header>interface.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>fps_checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>viewer</receiver>
   <slot>setFPSIsDisplayed(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
